components/brave_ads/browser/ads_service_impl.cc

```diff
@@ -697,6 +701,12 @@ void AdsServiceImpl::InitializeNotificationAdsPrefChangeRegistrar() {
       base::BindRepeating(&AdsServiceImpl::NotifyPrefChanged,
                           base::Unretained(this),
                           prefs::kMaximumNotificationAdsPerHour));
+  auto position_callback = base::BindRepeating(
```

Comment:
notifications_ad_position_callback

Reply:
i think this might be a bit overkill for a local var

Reply:
I would prefer more descriptive names even for local variables.